imports/plugins/core/ui/client/components/table/sortableTableComponents/pagination.js

```diff
@@ -23,13 +23,13 @@ class SortableTablePagination extends Component {
 
   getSafePage(page) {
     if (isNaN(page)) {
-      page = this.props.page; // eslint-disable-line
+      page = this.props.page;
```

Comment:
this throws a lint error because you shouldn't override function parameters. But why was it there and explicitly ignored? Confusing.